blobstore/api/clustermgr/volume.go

```diff
@@ -43,6 +43,8 @@ type Unit struct {
 type VolumeInfo struct {
 	Units []Unit `json:"units"`
 	VolumeInfoBase
+	// for scheduler
+	HasChunkCompacting bool `json:"has_chunk_compacting"`
```

Comment:
@JasonHu520 @Cloudstriff 
Since HasChunkCompacting is only used for schedulers, do we need to define a new structure here?